### ISSUE TITLE
Application (Flatpack) does not launch on Mint 20.x

### ISSUE THREAD

**Issue by RonnieTiger** [maltejur/discord-screenaudio#47]
```
ron@ron-desktop:~$ /usr/bin/flatpak run --branch=stable --arch=x86_64 --command=discord-screenaudio de.shorsh.discord-screenaudio
Gtk-Message: 18:33:42.727: Failed to load module "xapp-gtk3-module"
Qt: Session management error: None of the authentication protocols specified are supported
[default] QLayout: Attempting to add QLayout "" to StreamDialog "", which already has a layout
ron@ron-desktop:~$ 
```

**Comment by maltejur** [comment]
This could be https://github.com/Soundux/rohrkabel#%EF%B8%8F-problematic-distributions. Does the PPA mentioned there fix the issue?

**Comment by ronlaws86** [comment]
Testing on Mint 21 as well and it seems to hang with the same output in terminal, though it's not returning the prompt this with without ctrl-c, i'm making an assumption that a window is supposed to open? 
(sorry i did a fulll distro clean install since that bug report so can't confirm if the PPA will work, but Mint 21 is ubuntu 21 lts base i believe? )

**Comment by maltejur** [comment]
> i'm making an assumption that a window is supposed to open?

Yes that should indeed happen. Does it work any better when you directly build from source and don't use the flatpak?

**Comment by ronlaws86** [comment]
Compiling from source seems to open fine without issue, so this seems to be an issue with the flatpak version

**Comment by maltejur** [comment]
And the flatpak works with the PPA?
